great_expectations/datasource/fluent/interfaces.py

```diff
@@ -171,8 +172,10 @@ class DataAsset(FluentBaseModel, Generic[_DatasourceT]):
 
     order_by: List[Sorter] = Field(default_factory=list)
     batch_metadata: BatchMetadata = pydantic.Field(default_factory=dict)
+    batch_configs: List[BatchConfig] = Field(default_factory=list)
 
     # non-field private attributes
+    _save_batch_config: Callable[[BatchConfig], None] = pydantic.PrivateAttr()
```

Comment:
same comment as above, we want to call this from outside the scope of the asset, right?